contributing.md

```diff
@@ -68,11 +68,8 @@ features.add({
 	/** Whether to wait for DOM ready before runnin `init`. `false` makes `init` run right as soon as `body` is found. @default true */
 	waitForDomReady: false,
 
-	/** Whether to re-run `init` on pages loaded via AJAX. @default true */
-	repeatOnAjax: false,
```

Comment:
Want to give an example of using oneTime here?

Reply:
Thought about it, but it's not necessary